opam-ci-check/lib/opam_build.ml

```diff
@@ -122,10 +115,9 @@ let setup_repository ?(local=false) ~variant ~for_docker ~opam_version () =
      Otherwise the "opam pin" after the "opam repository set-url" will fail (cannot find the new package for some reason) *)
   run "%s -f %s/bin/opam-%s %s/bin/opam" ln prefix opam_version_str prefix ::
   run ~network "opam init --reinit%s -ni" opamrc :: (* TODO: Remove ~network when https://github.com/ocurrent/ocaml-dockerfile/pull/132 is merged *)
-  run "uname -rs && opam exec -- ocaml -version && opam --version" ::
+  run "%sopam config report" (match opam_version with `V2_1 | `V2_2 | `V2_3 | `Dev -> "opam option solver=builtin-0install && " | `V2_0 -> "") ::
```

Comment:
I'm assuming this implies you've tested to confirm that this works on opam 2.1!

Reply:
Yes, opam 2.1 onwards supports `opam option`, and 0install is already [enabled](https://github.com/ocurrent/ocaml-dockerfile/blob/dc5ab758fd581ac55b340af6060a7552d50200ba/src-opam/opam.ml#L242) at compile time.